productivity-suite/app/fragments/news/src/root.css

```diff
@@ -0,0 +1,300 @@
+piercing-fragment-host[fragment-id="news"] {
```

Comment:
Note: I'm disregarding the css here as I plan to tweak it afterwards (since @jculvey gave me the carte blanche 😍)